hooks/post_gen_project.py

```
@@ -45,19 +45,23 @@ def init_proto():
 
-    print("Step 2/4: Running 'make generate'...")
+    print("Step 2/5: Running 'make generate'...")
     code = Popen(["make", "generate"], cwd=PROJECT_DIRECTORY).wait()
     if code != 0:
         print("Error: 'make generate' failed.")
         sys.exit(code)
 
-    print("Step 3/4: Running 'make mock'...")
+    print("Step 3/5: Tidying Go modules (pre-mock)...")
+    # -e flag ignores errors from missing mock packages (generated next step)
+    Popen(["go", "mod", "tidy", "-e"], cwd=PROJECT_DIRECTORY).wait()
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)